keyboards/kprepublic/bm40hsrgb/keymaps/34keys/rules.mk

```diff
@@ -0,0 +1,4 @@
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes # Mouse keys
+COMBO_ENABLE = yes  #  The firmware is too large!
+EXTRAKEY_ENABLE = yes # Audio control and System control
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = yes
MOUSEKEY_ENABLE = yes
COMBO_ENABLE = yes
EXTRAKEY_ENABLE = yes
```

Reply:
remove that comments you requested. Thanks!